tests/features/multisite_wysiwyg/rich_text_editing.feature

```diff
@@ -0,0 +1,26 @@
+@api
+Feature: Testing the rich text options available with the toolbar present on
+  WYSIWYG fields like "Body".
+  I want to be able to format my content with rich text options supplied by the
+  displayed toolbar
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
```

Comment:
No that could be another role but this one for this specific test ensures the scenario is valid even if evolutions occur in content editing permissions or workflows.